src/OpenTelemetry.Extensions/README.md

```diff
@@ -13,3 +13,25 @@ future.
 
 Adds a log processor which will convert log messages into events and attach them
 to the currently running Activity.
+
+## Traces
+
+### AutoFlushActivityProcessor
+
+Processor that flushes its containing TracerProvider if an ended activity matches a predicate. 
+Note that this processor must be added *after* exporter related span processors.
+
+Example of AutoFlushActivityProcessor usage:
+```cs
+public static TracerProviderBuilder AddMyExporter(this TracerProviderBuilder builder, MyExporterOptions options)
+{
+    var exporter = new MyExporter(options);
+
+    var autoFlushProcessor = new AutoFlushActivityProcessor(
+        a => a.Parent == null && (a.Kind == ActivityKind.Server || a.Kind == ActivityKind.Consumer), 5000);
+
+    return builder
+        .AddProcessor(new SimpleActivityExportProcessor(exporter))
```

Comment:
Would using the BatchActivityExporterProcessor in the example be more interesting? Using the Simple one, more than likely the flush afterwards would not have anything to flush.

Reply:
+1

Reply:
Updated example.